core/src/main/java/org/mule/runtime/core/internal/processor/AsyncDelegateMessageProcessor.java

```diff
@@ -90,31 +106,62 @@ public void initialise() throws InitialisationException {
     } else {
       processingStrategy = DIRECT_PROCESSING_STRATEGY_INSTANCE;
     }
-    if (delegate == null) {
+    if (delegateBuilder == null) {
       throw new InitialisationException(objectIsNull("delegate message processor"), this);
     }
+
+    delegateBuilder.setProcessingStrategy(processingStrategy);
+    delegate = delegateBuilder.build();
+    initialiseIfNeeded(delegate, muleContext);
+
     super.initialise();
   }
 
   @Override
   public void start() throws MuleException {
-    scheduler = schedulerService
-        .ioScheduler(muleContext.getSchedulerBaseConfig().withName(name != null ? name : getLocation().getLocation()));
-    reactorScheduler = fromExecutorService(scheduler);
+    if (ownProcessingStrategy) {
+      startIfNeeded(processingStrategy);
+    }
+
+    sink = processingStrategy
```

Comment:
This ownProcessingStrategy isn't invalid, but is it necessary? Or is this only needed to support custom processing strategy necessary?

Reply:
yes, for allow setting a `maxConcurrency` in the async (next PR to come)